### ISSUE TITLE
blacklist add changes the path to add

### ISSUE THREAD

**Issue by max-phase** [robbert-vdh/yabridge#156]
### Thanks for giving yabridge a shot!

- [X] I read through both the troubleshooting and the known issues sections, and my issue wasn't listed there

### Problem description

Blacklist seems to add 'canonical' paths:

```
$ yabridgectl list
/mnt/music/music/vst3
/mnt/music/music/vst64
$ yabridgectl blacklist add /mnt/music/music/vst64/Guitar\ Rig\ 5.dll
$ yabridgectl blacklist list
/mnt/win/data/music/vst64/Guitar Rig 5.dll
```

### What did you expect to happen?

yabridgectl sync to *not* add a '.so' for Guitar Rig 5

### What actually happened?

yabridgectl sync adding a '.so' for Guitar Rig 5

### Operating system

5.15.11-gentoo-x86_64

### How did you install yabridge?

build from master dabefef098b4ef37a1e3d2

### yabridge version

dabefef098b4ef37a1e3d2

### yabridgectl version

dabefef098b4ef37a1e3d2

### Wine version

wine-staging-6.23

### Plugin

Guitar Rig 5

### Plugin type

VST2

### Plugin architecture

64-bit

### Host

n.a.

**Comment by robbert-vdh** [comment]
Thanks for reporting, fixed in 263b192daa316f801fba87055555ebcb1c348c44.

**Comment by max-phase** [comment]
wow - that was quick!
Confirmed - fixed!
..I should come up with something more difficult.... ;)
